tree/tree/Controller/Live/LiveViewController.swift

```diff
@@ -0,0 +1,264 @@
+//
+//  LiveViewController.swift
+//  tree
+//
+//  Created by ParkSungJoon on 29/01/2019.
+//  Copyright © 2019 gardener. All rights reserved.
+//
+
+import UIKit
+import Intents
+import NetworkFetcher
+
+class LiveViewController: UIViewController {
+    
+    @IBOutlet weak var scrollView: UIScrollView!
+    @IBOutlet weak var pageControl: UIPageControl!
+
+    private var livePagerPages: [UIView] = []
+    private var googleTrendData: TrendDays? {
+        didSet {
+            guard let trendPage = livePagerPages.first as? TrendPageView else { return }
+            trendPage.googleTrendData = googleTrendData
+        }
+    }
+    private var recentEventData: [CategoryEvents]? {
+        didSet {
+            guard let eventPage = livePagerPages.last as? EventPageView else { return }
+            eventPage.recentEventData = recentEventData
+        }
+    }
+    private var countryName: String = Country.usa.info.name
+    private var loadingView: LoadingView?
+    private let localizedLanguage = LocalizedLanguages.korean.rawValue
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        scrollView.contentInsetAdjustmentBehavior = .never
+        setupTrendPages()
+        fetchDailyTrends(from: Country.usa.info.code)
+        fetchTopRecentEvents()
+        addNotificationObserver()
+    }
+    
+    @objc func receiveCountryInfo(_ notification: Notification) {
+        guard let countryInfo = notification.userInfo as? [String: String] else { return }
+        guard
+            let countryCode = countryInfo["code"],
+            let countryName = countryInfo["name"]  else {
+                return
+        }
+        fetchDailyTrends(from: countryCode)
+        self.countryName = countryName
+    }
+    
+    private func addNotificationObserver() {
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(receiveCountryInfo(_:)),
+            name: .observeCountryChanging,
+            object: nil
+        )
+    }
+    
+    // MARK: - Intent Setting
+    private func donate(country: String) {
+        let intent = TrendIntent()
+        intent.suggestedInvocationPhrase = "\(countryName) 급상승 검색어"
+        intent.country = country // 급상승 검색어를 불러올 국가를 선택합니다.
+        
+        let interaction = INInteraction(intent: intent, response: nil)
+        interaction.donate { (error) in
+            if error != nil {
+                if let error = error as NSError? {
```

Comment:
`as?` 다운캐스팅 문법을 이용하면 위 중첩 if문을 하나로 합칠 수 있습니다. (https://docs.swift.org/swift-book/LanguageGuide/TypeCasting.html)



```suggestion
if let error = error as? NSError {
}
```